src/lib/footer-navigation/FooterNavigation.svelte

```diff
@@ -9,6 +10,9 @@
   import FooterBottomBar from './FooterBottomBar.svelte';
   import styles from './FooterNavigation.module.scss'
 
+  const ctx = getAppContext()
+  $: ({minFooter} = $ctx.toolConfig)
```

Comment:
Maybe this is a Svelte thing, but can you explain what this line does?

Reply:
Yes, it's all svelte syntax.

It destructures toolConfig - this one is somewhat visible. but the `$:` makes the expression "Reactive" - it updates every time any of the elements change. Basically an equivalent of `useEffect` in react.

Reply:
Thanks for the education!